tools/bin/gh_action_zombie_killer

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# ------------- Import some defaults for the shell
+
+# Source shell defaults
+# $0 is the currently running program (this file)
+this_file_directory=$(dirname $0)
+relative_path_to_defaults=$this_file_directory/../shell_defaults
+
+# if a file exists there, source it. otherwise complain
+if test -f $relative_path_to_defaults; then
+  # source and '.' are the same program
+  source $relative_path_to_defaults
+else
+  echo -e "\033[31m\nFAILED TO SOURCE TEST RUNNING OPTIONS.\033[39m"
+  echo -e "\033[31mTried $relative_path_to_defaults\033[39m"
+  exit 1
+fi
+
+echo "To run locally use GITHUB_PAT=\$YOUR_PAT_HERE before running"
+token=$GITHUB_PAT
+org=airbytehq
+# FUN POSIX fact, every string is an array!
+repo_list="airbyte airbyte-cloud"
```

Comment:
commenting to make sure the github service account has permissions to airbyte cloud repo